src/App.js

```diff
@@ -1,21 +1,41 @@
 import React from 'react';
 import logo from './logo.svg';
-import './App.css';
+import './components/page.css';
+import Header from './components/header';
+import Home from './components/home';
+import News from './components/news';
+import Schedule from './components/schedule';
+import About from './components/about';
+import Article from './components/article'
 
-import {NavBars, News, Schedule} from './page';
+import { BrowserRouter, Switch, Route } from 'react-router-dom';
 
 function App() {
   return (
 
-    <div className="container">
-      <NavBars/>
-      <div className="content">
-        <News/>
-        <Schedule/>
-      </div>
-    </div>
+  	<BrowserRouter>
+	    <div className="container">
+	    
+	      	<Header/>
+
+			<Switch>
+				<Route exact path='/' component={Home} />
+				<Route exact path='/news' component={News} />
+				<Route exact path='/schedule' component={Schedule} />
+				<Route exact path='/about' component={About} />
+				<Route exact path='/:game' render={(props) => (
+					<News game={props.match.params.game}/>
+					)}/>
+				<Route exact path='/:game/:articleLink' render={(props) => (
+					<Article game={props.match.params.game} articleLink={props.match.params.articleLink}/>
```

Comment:
You can also just pass article instead of articleLink.
Also in current implementation you don't really need using renderProps, because you can just use `component={Article}` here, and get article and articleLink from params inside of component.